src/components/MenuButton/src/bem-mapping.jsx

```diff
@@ -0,0 +1,7 @@
+export const menuButtonClasses = {
+  root: 'denhaag-menubutton',
```

Comment:
`denhaag-menu-button` is nicer imo. We've used hyphens where the component itself would have PascalCasing in most places

EDIT: Why is there a bem-mapping file if there's no MUI classes we need to overwrite?